synapseAnnotations/data/experimentalData.json

```diff
@@ -1361,6 +1381,11 @@
         "value": "Rhesus macaque",
         "description": "Macaca mulatta with taxonomy ID: 9544 and Genbank common name: Rhesus monkey",
         "source": "https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=9544&lvl=3&p=has_linkout&p=blast_url&p=genome_blast&p=mapview&lin=f&keep=1&srchmode=1&unlock"
+      },
+      {
+        "value": "Chicken",
```

Comment:
Would instead suggest gallus gallus. A preferred source for these is probably the NLM taxonomy browser (given the other entries).